docs/LabeledRange.mdx

```diff
@@ -30,8 +30,8 @@ All `LabeledRange` props are passed to the underlying [Slider](#/docs-slider) co
     defaultValue={[1, 100]}
     disabled={false}
     label="Height"
-    formatter={(min, max, maxIndicator) => {
-      return "" + min + " - " + max + maxIndicator + " in" 
+    formatter={(min, max, unit, maxIndicator) => {
```

Comment:
I'm wondering if it might be clearer to either use the `unit` or the `formatter` prop, but not both.

The `unit` prop can be used as a convenient way of specifying a unit with standard formatting. If you're passing in a `formatter` prop, I think that function can handle the unit displayed as well.

Reply:
[Talked IRL - definitely makes sense, adding a fresh commit to implement]